Game/enc_temp_folder/7ce345a7f675f6fd8b5ac5ecee8c345/BlackboardValues.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Components/ActorComponent.h"
+#include "BlackboardValues.generated.h"
+
+USTRUCT()
+struct GDKTESTGYMS_API FBlackboardValues 
+{
+	GENERATED_BODY()
+
+	UPROPERTY()
+	FVector TargetAValue; // Points to run betweeen
+	UPROPERTY()
+	FVector TargetBValue;
+	UPROPERTY()
+	bool TargetStateIsAOrB; // Keep track of current goal target, to continue towards it after handover. True for A, false for B
```

Comment:
nit: aligns more with comment :D 
```suggestion
	bool TargetStateIsA; // Keep track of current goal target, to continue towards it after handover. True for A, false for B
```